content/docs/csidriver/features/powermax.md

```diff
@@ -399,7 +399,7 @@ After a successful installation of the driver, if a node Pod is running successf
 
 The values for all these keys are always set to the name of the provisioner which is usually `csi-powermax.dellemc.com`.
 
-> *NOTE:* The Topology support does not include any customer-defined topology, that is, users cannot create their own labels for nodes and storage classes and expect the labels to be honored by the driver.
+Starting from version 2.3.0, topology keys has been enhanced to filter out arrays, and associated transport protocol available to each node and create topology keys based on any such user input.
```

Comment:
Change "has been enhanced" to "have been enhanced". 
Delete "and" before "associated

Reply:
addressed